cli/command/cli.go

```diff
@@ -92,6 +92,8 @@ type DockerCli struct {
 	// this may be replaced by explicitly passing a context to functions that
 	// need it.
 	baseCtx context.Context
+
+	enableGlobalMeter, enableGlobalTracer bool
```

Comment:
nit: we should probably call these Meter/Tracer**Provider**

Reply:
Yeah, they were already far too  long names for an un-exported field 😂